docs/prod-host-setup.md

```diff
@@ -167,3 +167,35 @@ Verify that swap is disabled by running:
 ```bash
 grep -q "/dev" /proc/swaps && echo "swap partitions present (Recommendation: no swap)" || echo "no swap partitions (OK)"
 ```
+
+#### Use a kernel that is patched for [CVE-2019-3016](https://nvd.nist.gov/vuln/detail/CVE-2019-3016)
```

Comment:
I don't know if we should put this here. This is a transitory state, until the fix for this issue gets merged in stable kernels and in long term support distros.
As a security practice anyone should run on updated kernels.

I would just post a security notice.